File: app/src/main/java/com/android/shaheer/recording/model/RecordItem.java
package com.android.shaheer.recording.model;

import com.android.shaheer.recording.utils.Player;

public class RecordItem {

    private String recordAddress;
    public String getRecordAddress() {
        return recordAddress;
    }

    private String recordExtension;
    public String getRecordExtension() { return recordExtension; }

    private String recordDuration;
    public String getRecordDuration() {
        return recordDuration;
    }

    private boolean isSelected = false;
    public boolean isSelected() { return isSelected; }
    public void setSelected(boolean selected) { isSelected = selected; }


    public RecordItem(String recordAddress, String recordExtension, String recordDuration) {
        this.recordAddress = recordAddress;
        this.recordExtension = recordExtension;
        this.recordDuration = recordDuration;
    }

    public RecordItem copy(){
        RecordItem record = new RecordItem(this.recordAddress, this.recordExtension, this.recordDuration);
        record.isSelected = this.isSelected;
        return record;
    }
}

File: app/src/main/java/com/android/shaheer/recording/model/RecordItem.kt
package com.android.shaheer.recording.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
data class RecordItem(
    val recordAddress: String,
    val recordExtension: String,
    val recordDuration: String,
    var isSelected: Boolean = false
): Parcelable {
}

/*
public RecordItem copy(){
        RecordItem record = new RecordItem(this.recordAddress, this.recordExtension, this.recordDuration);
        record.isSelected = this.isSelected;
        return record;
    }
* */